testing/py3-blis/APKBUILD

```diff
@@ -0,0 +1,41 @@
+# Contributor: Oleg Titov <oleg.titov@gmail.com>
+# Maintainer: Oleg Titov <oleg.titov@gmail.com>
+pkgname=py3-blis
+pkgver=0.4.1
+pkgrel=0
+pkgdesc="Fast matrix-multiplication as a self-contained Python library"
+url="https://github.com/explosion/cython-blis"
+arch="x86_64"
+license="MIT"
+depends="python3"
+makedepends="python3-dev py3-setuptools cython3"
```

Comment:
should be cython

Reply:
Done. Thank you! Should I apply the same to `testing/py3-cymem` and change `cython3` to `cython`?